site/index.html

```diff
@@ -76,6 +76,62 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>A new paradigm</h3>
+            <p>Express your ideas clearly and learn a new way of thinking about programming. Based on lambda calculus Haskell is a
+            purely functional programming language that features referential transparency, immutability and lazy evaluation.
+            Concepts that will blow your mind
+               &mdash; relearn programming while having an absolute blast.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Composition and predictability</h3>
+            <p>Reason about large pieces of code and compose them easily. There is no global state
+            or mutable variables obscuring the meaning of your program. The strong type system makes sure there are no surprises
+               &mdash; never again will you have to guess what your program does at execution time.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+           <h3>Declarative</h3>
+           <p>Write your programs declaratively by utilizing the power of pure functions and algebraic data types.
+              In Haskell we don't write how a program should be executed, we just describe its logic
+                 &mdash; never again be forced to think about evaluation order or execution details.
+           </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Performance</h3>
+            <p>Squeeze out the last ticks of your multi-core processors, thanks to best-in-class support for async,
+            concurrent and parallel programming... made possible via garbage collection and green threads.
```

Comment:
the "... made possible via" feels grammatically weird. Just "parallel programming...This is made possible by" would be better, I feel

Reply:
Starting another sentence breaks the flow. We could replace "via" with "by" though. Not sure that makes much of a difference.

Reply:
Then I'd just avoid the "..." and use a comma.